webapp/app.py

```diff
@@ -67,7 +68,7 @@ def _make_bar_graph(version_summaries, title, threshold, key_color_map):
                 percentage = value / getattr(aggregate_version_summary, measure)
                 data_vectors['percentage'].append(percentage)
 
-                data_vectors['label'].append('{:.2f}%<br>({:,.0f})'.format(percentage * 100, value))
+                data_vectors['label'].append('{:.2f}% ({:,.0f})'.format(percentage * 100, value))
```

Comment:
please revert - this fixes the bar chart label rendering issues

Reply:
Yep, this one should be resolved in [e311f8b](https://github.com/symbol/nodewatch/pull/3/commits/e311f8bf5d4db8ae80eb5cd0ed6984b63f58f685).